invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -94,6 +94,22 @@ def record_version(obj):
     return field_data
 
 
+def mask_removed_by(obj):
+    """Mask information about who removed the record."""
+    return_value = "Unknown"
+    removed_by = obj.get("removed_by", None)
+
+    if removed_by is not None:
+        user = removed_by.get("user", None)
+
+        if user == "system":
+            return_value = "System (automatic)"
```

Comment:
all the returned values are missing translations, unless I misunderstood this serializer and it is done somewhere else

Reply:
to be extra sure, does flask-babel return the resolved value as a string-to-translate when we extract messages?